account/accounts.go

```diff
@@ -365,6 +370,53 @@ func (m *Manager) Accounts() ([]gethcommon.Address, error) {
 	return filtered, nil
 }
 
+// NewOnboarding starts the onboarding process generating accountsCount accounts and returns a slice of OnboardingAccount.
+func (m *Manager) NewOnboarding(accountsCount, mnemonicPhraseLength int) ([]*OnboardingAccount, error) {
```

Comment:
Any reason not to do:

```suggestion
func (m *Manager) SetOnboarding(onboarding *Onboarding) {
```